### ISSUE TITLE
Doctors lack a way to treat pain, or at least know about it

### ISSUE THREAD

**Issue by Jolly-66** [TaleStation/TaleStation#3346]
This is just a little house keeping for me, and to make the pain system a little more obvious to new players.
